readme.md

```diff
@@ -13,6 +13,7 @@ Missing something awesome? Anyone is welcome to [submit projects to this list](h
 * [nvim-dvc](https://github.com/gennaro-tedesco/nvim-dvc): Neovim plugin for DVC.
 * [COVID Genomics/Airflow-DVC](https://github.com/covid-genomics/airflow-dvc): Airflow extension for DVC.
 * [COVID Genomics/dvc-fs](https://github.com/covid-genomics/dvc-fs): High-level abstraction for DVC file manipulation (listing & I/O) with basic support for [PyFilesystem2](https://github.com/PyFilesystem/pyfilesystem2).
+* [ZnTrack](https://github.com/zincware/ZnTrack): Object-Relational Mapping for https://dvc.org in Python.
```

Comment:
```suggestion
* [ZnTrack](https://github.com/zincware/ZnTrack): Create, visualize, benchmark & run DVC pipelines in Python & Jupyter notebooks.
```

Reply:
I like it! Sounds less technical and more appealing.

Reply:
Unfortunately I can't commit the changes.

Reply:
buggy GH interface, I've had similar issues too.

Reply:
Oh I see you updated the upstream description too :heart: ![](https://img.shields.io/badge/dynamic/json?label=zincware/ZnTrack&query=%24.description&url=https%3A%2F%2Fapi.github.com%2Frepos%2Fzincware%2FZnTrack&logo=GitHub)